### ISSUE TITLE
New firmware today

### ISSUE THREAD

**Issue by StanLee4456** [brett8883/DJI_Super-Patcher#74]
Hey Brett- there was a new firmware today to improve some camera settings on the P4P2. 

Can you upload another file to make this compatible? As a Xmas gift?
Thanks so much.

**Comment by knarfboy** [comment]
the v01.00.50.00 has a anti rollback trap

**Comment by StanLee4456** [comment]
True. But as long as SP removes the alt + NFZ restrictions I don’t care.

**Comment by knarfboy** [comment]
@ Brett8883,
I see that you have little time, but I would like to ask you to patch a current bin file for the P4Pv2.0 for us.
Merry Christmas

**Comment by StanLee4456** [comment]
Brett- you still around these parts?

**Comment by knarfboy** [comment]
digdat0 made it:
https://github.com/digdat0/DJI_FC_Patcher/blob/master/FC_patch_sequence_for_dummy_verify.sh

**Comment by StanLee4456** [comment]
> digdat0 made it:
> https://github.com/digdat0/DJI_FC_Patcher/blob/master/FC_patch_sequence_for_dummy_verify.sh

I tried using that as the firmware with superpatcher. It doesn’t seem to work.

**Comment by knarfboy** [comment]
dummy_verify is not a patched Firmware!

https://dji.retroroms.info/howto/manually_installing_custom_fw_from_fc_patcher

**Comment by StanLee4456** [comment]
Sorry- he had sent me a bin file. It didn’t work.

**Comment by StanLee4456** [comment]
> dummy_verify is not a patched Firmware!
> https://dji.retroroms.info/howto/fc_patcher_tool
> https://dji.retroroms.info/howto/manually_installing_custom_fw_from_fc_patcher
> 
> Best wait for Brett8883 update super patcher...

Hopefully it won’t take too long. Maybe a New Year’s gift?

**Comment by knarfboy** [comment]
what bin file he had send to you?

**Comment by StanLee4456** [comment]
https://drive.google.com/file/d/1ZlX9QGG18axYVZCDoEbo1L1ZCMBw3eFx/view?usp=sharing

**Comment by StanLee4456** [comment]
But the bin file doesn’t work.

**Comment by knarfboy** [comment]
@brett8883 ,
can you create a v01.00.50.00 bin file? I will test it.

**Comment by StanLee4456** [comment]
@brett8883 ,
I’ll happily do testing as well.

**Comment by knarfboy** [comment]
If P4P2.0 is upgraded to 01.00.50.00 firmware, it has rejected dumldore's adb function.
